source/User_Guide/email_activity_beta.md

```diff
@@ -21,33 +22,100 @@ The Email Activity feed allows you to view a snapshot of email-related activity
 
 With the Email Activity feed you can:
 
-* Troubleshoot email delivery issues faster with comprehensive and sequential event data.
-* Pinpoint specific emails easily with advanced search and filter options -- including subject line and email metadata.
+* Troubleshoot email delivery issues faster with comprehensive and sequential event data by email.
+* Pinpoint specific emails easily with advanced search and filter options --including subject line and email metadata.
 * Customize your data independently with CSV download/export option.
 * Increase storage available for up to 60 days for historical visibility within the UI by purchasing an upgrade.
-* Complete an API call to gather specific email events when additional storage is purchased.
+* Call all events by an email from the API with additional storage purchase.
 
 {% info %}
-SendGrid stores two days of Email Activity data for free. To increase your storage capability, see LINK
+SendGrid stores two days of Email Activity data for free. To increase your storage capability, you may purchase more storage [here](https://app.sendgrid.com/settings/billing).
 {% endinfo %}
 
-{% anchor h2 %}
+{% anchor H2 %}
+Types of email activity data
+{% endanchor %}
+
+The Email Activity feed allows you to view specific information about messages sent from your account. 
+
+
+<table class="table" style="table-layout:fixed">
+  <tr>
+  <th class="tg-031e">Event</th>
+    <th class="tg-yw4l">Description</th>
+    </tr>
+    <tr>
+    <td class="tg-031e">Processed</td>
+    <td class="tg-yw4l">Requests from your website, application, or mail client via SMTP Relay or the API that SendGrid processed.</td>
+  </tr>
+  <tr>
+    <td class="tg-031e">Clicks</td>
+    <td class="tg-yw4l">Whenever a recipient clicks one of the Click Tracked links in your email.</td>
+  </tr>
+  <tr>
+    <td class="tg-031e">Delivered</td>
+    <td class="tg-yw4l">An email that was delivered to a recipient.</td>
```

Comment:
Passive. How about:

The accepted response generated by the recipients' mail server.